### PR TITLE
feat: skeleton code for subscriber and ipfs

### PR DESCRIPTION
**Pull request by inchori**
Add skeleton code for Panacea event subscriber.

### CODE REVIEW

File: event/expected.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package event
+
+import (
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/medibloc/panacea-oracle/config"
+	"github.com/medibloc/panacea-oracle/panacea"
+	"github.com/medibloc/panacea-oracle/sgx"
+)
+
+// Reactor contains all ingredients needed for handling this type of event
```
Comment (youngjoon-lee) [inline event/expected.go:10]:
```suggestion
// Reactor contains all ingredients needed for handling all event types
```
IIRC

File: panacea/grpc_client.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+type GrpcClient struct {
+	conn *grpc.ClientConn
+}
+
+func NewGrpcClient(grpcAddr string) (*GrpcClient, error) {
```
Comment (youngjoon-lee) [inline panacea/grpc_client.go:19]:
```suggestion
type GRPCClient struct {
	conn *grpc.ClientConn
}

func NewGRPCClient(grpcAddr string) (*GRPCClient, error) {
```

**Review by youngjoon-lee — APPROVED** [review]
lgtm

File: event/expected.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+// Reactor contains all ingredients needed for handling all event types
+type Reactor interface {
+	GRPCClient() *panacea.GRPCClient
+	EnclaveInfo() *sgx.EnclaveInfo
+	OracleAcc() *panacea.OracleAccount
+	OraclePrivKey() *btcec.PrivateKey
+	Config() *config.Config
+	QueryClient() *panacea.QueryClient
+	Ipfs() *ipfs.Ipfs
```
Comment (youngjoon-lee) [inline event/expected.go:19]:
```suggestion
	IPFS() *ipfs.IPFS
```

File: ipfs/ipfs.go [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	shell "github.com/ipfs/go-ipfs-api"
+	log "github.com/sirupsen/logrus"
+)
+
+type Ipfs struct {
+	sh *shell.Shell
+}
+
+// NewIpfs generates an ipfs node with ipfs url.
+func NewIpfs(url string) *Ipfs {
```
Comment (youngjoon-lee) [inline ipfs/ipfs.go:15]:
```suggestion
type IPFS struct {
	sh *shell.Shell
}

// NewIPFS generates an IPFS shell (API client) with IPFS url.
func NewIPFS(url string) *IPFS {
```

**Review by youngjoon-lee — APPROVED** [review]

**Review by H4NLee — APPROVED** [review]
lgtm

**Pull request merged**